web/src/screens/filters/action.tsx

```diff
@@ -400,6 +400,14 @@ const TypeForm = ({ action, idx, clients }: TypeFormProps) => {
               />
             </div>
           </div>
+          <div className="col-span-12 sm:col-span-6">
+            <div className="col-span-6">
+              <SwitchGroup
+                name={`actions.${idx}.paused`}
+                label="Don't start download automatically"
```

Comment:
```suggestion
                label="Add paused"
                description="Add torrent as paused"
```
To make it uniform with the rest of the clients.

Reply:
I think it's fine - they're matching the ruTorrent name which is what users at this level would expect. Think of it like a translation string.